### ISSUE TITLE
Comment 3

### ISSUE THREAD

**Issue by ak2253** [NeverFlight/CS435-Project2-Part1#3]
Consider replacing the boolean array visited with a list/arraylist visited in the following method:
https://github.com/NeverFlight/CS435-Project2-Part1/blob/94014a578e3e818e3883b7d84039f623c8d42454/src/com/cs435/part1/Main.java#L81-L102
This way you can use the contain method so that it is not dependent on how the graph is formatted in case a graph where it has 5 nodes where each are double digits in value is passed into the method.

**Comment by NeverFlight** [comment]
I agree with you. But under this project, I think that's fine when I only define all nodes' value will be on the range from 0 to n and they all are integer datatype.
BTW, if the value is a double datatype,  even though I use the ArrayList or HashSet, I still cannot handle this situation if I define they all can only contain the Integer datatype.
